src/python/pants/goal/run_tracker.py

```diff
@@ -81,14 +84,21 @@ def start(self, run_start_time: float, specs: List[str]) -> None:
 
         # Initialize the run.
         self._run_start_time = run_start_time
-        self.run_info.add_basic_info(self.run_id, run_start_time)
-        cmd_line = " ".join(["pants"] + sys.argv[1:])
-        self.run_info.add_info("cmd_line", cmd_line)
-        self.run_info.add_info("specs_from_command_line", specs, stringify=False)
 
-        # Create a 'latest' symlink, after we add_infos, so we're guaranteed that the file exists.
-        link_to_latest = os.path.join(os.path.dirname(self.run_info_dir), "latest")
-        relative_symlink(self.run_info_dir, link_to_latest)
+        self._run_info["id"] = self.run_id
+        self._run_info["timestamp"] = run_start_time
+
+        datetime = time.strftime("%A %b %d, %Y %H:%M:%S", time.localtime(run_start_time))
+        self._run_info["datetime"] = datetime
+
+        self._run_info["user"] = getpass.getuser()
+        self._run_info["machine"] = socket.gethostname()
+        self._run_info["buildroot"] = get_buildroot()
+        self._run_info["version"] = VERSION
+
+        cmd_line = " ".join(["pants"] + sys.argv[1:])
+        self._run_info["cmd_line"] = cmd_line
+        self._run_info["specs_from_command_line"] = specs
```

Comment:
Nit: it's simpler (and likely more performant?) to use .update() and pass a single dict, rather than each of these individual synchronous calls.